src/pages/layer-2.js

```diff
@@ -406,19 +406,37 @@ const Layer2Page = ({ data }) => {
         </StatsContainer>
       </PaddedContent>
 
+      <PaddedContent id="what-is-layer-2">
+        <h2>What is layer 2?</h2>
+        <p>
+          Layer 2 (L2) is a collective term to describe a specific set of
```

Comment:
Regarding copy for defining L2, I very much like Barnabé's explanation:
https://barnabe.substack.com/p/understanding-rollup-economics-from?s=r

He's speaking specifically to rollups but I like how he frames them as an extension the protocol they build on. I think saying L2s extend Ethereum is better than merely saying they're connected to Ethereum.